app/controllers/invitation_controller.rb

```diff
@@ -60,11 +66,13 @@ def update_join_team_request(user, student)
     end
   end
 
+  #auto complete user name
```

Comment:
we thought in the last function, line 51, when other possibilities are eliminated, this function could be implemented then.